backend/typescript/rest/camperRoutes.ts

```diff
@@ -23,6 +23,9 @@ const camperRouter: Router = Router();
 
 const camperService: ICamperService = new CamperService();
 
+// TODO: secure stripe keys
+const STRIPE_ENDPOINT_KEY = process.env.STRIPE_ENDPOINT_SECRET || "";
```

Comment:
are we going to implement request verification in another PR or is this a TODO item for this one? cuz from my understanding we need to use this to verify that the requests we receive at the webhook endpoint are legit for security

Reply:
Yeah, we'll need to do both. I just put it as its own PR